AndroidSDKTests/src/test/java/com/leanplum/_whitebox/utilities/ResponseHelper.java

```diff
@@ -74,4 +74,19 @@ public static void seedResponse(String filename) {
       Log.e("ResponseHelper", "Unable to seed response from file: " + filename);
     }
   }
+
+  /**
+   * Seeds the Null response to Util.getResponse method.
+   *
+   */
+
+
+  public static void seedResponseNull(String filename) {
```

Comment:
don't need a file here since you're only setting `null` as the response

Reply:
done